src/Files.App/Actions/Open/OpenContextMenuAction.cs

```diff
@@ -0,0 +1,86 @@
+﻿using CommunityToolkit.WinUI.UI;
+using Files.App.Commands;
+using Files.App.Extensions;
+using Files.App.Filesystem;
+using Files.App.UserControls;
+using Files.App.UserControls.Widgets;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Input;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class OpenContextMenuAction : IAction
+	{
+		public string Label { get; } = "OpenContextMenu".GetLocalizedResource();
+		public string Description => "TODO: Need to be described.";
+
+		public HotKey HotKey { get; } = new(VirtualKey.Enter, VirtualKeyModifiers.Control);
```

Comment:
Can you set the default to the `menu` key?

Reply:
It's done.